meetings/2021-11-17.md

```diff
@@ -0,0 +1,189 @@
+* Repository: https://github.com/solid/authorization-panel
+
+
+## Present
+* Elf Pavlik
+* Aaron Coburn
+* Justin B
+* Henry S
+* Matthieu B
+
+## Regrets
+* 
+
+---
+
+## Announcements
+
+### Meeting Recordings and Transcripts
+* No audio or video recording, or automated transcripts without consent. Meetings are transcribed and made public. If consent is withheld by anyone, recording/retention must not occur.
+* Use panel chat and repository. Queue in call to talk.
+
+
+### Participation and Code of Conduct
+* [Join the W3C Solid Community Group](https://www.w3.org/community/solid/join), [W3C Account Request](http://www.w3.org/accounts/request), [W3C Community Contributor License Agreement](https://www.w3.org/community/about/agreements/cla/)
+* [Solid Code of Conduct](https://github.com/solid/process/blob/master/code-of-conduct.md), [Positive Work Environment at W3C: Code of Ethics and Professional Conduct](https://www.w3.org/Consortium/cepc/)
+* If this is your first time, welcome! please introduce yourself.
+
+
+### Scribes
+* Henry S
+* e Pavlik
+
+
+### Introductions
+* name: text
+*
+
+---
+
+## Topics
+
+### Questions about graphMetrix implementation
+
+* **Frederick Gibson (Fred)**: I am the CEO of Graphmetrix. 
+ We have been around for a couple of years
+ and provide Solid Pod server and Apps
+ Our App is a productivity app suite (similar to Zoho(?))
+ We have a digital twin system inside the pod service.
+ We of course try to be 100% compatible with Solid to interoperability.
+* **Elf**: We discussed authz related topics yesterday and your implementation came up
+* **JustinB: The main question that myself and Barath were interested in was**:
+   Barath was under the impression that you don't have / URI semantics, ie containers end in `/`.
+* **Frederick Gibson**: We fully support the solid spec. 
+   Every container in our system has permanent URI, so that the same resource can be in multiple containers.
+* **JustinB**: ah, that is good to know.  
+  The context of the discussion was not just about this, but also about authorization and how you deal with access control and inheritance and containment. 
+* **Fred**: 
+  The current WAC system does not work for us. The problem is copying the data around. ...
+* **Henry**: Which issue are you talking about? Can you put link in our notes. 
+* **Fred: https**://github.com/solid/web-access-control-spec/issues/102
+* **Pavlik**: Based on what do you apply inheritance? 
+  The URI / paths or ldp:contains statements.
+* **Fred**: ...
+* **Henry**: We discussed this in issue 102 a bit, I'm all for changing WAC where we can to fix problems, rather than re-inventing the whole thing.
+* ...: I definitely see that having a default rule is very useful, as it avoids copying rules all over the place.
+* ...: I see a problem with requiring the user to be logged before the server can determine if you the app can see something an ACL or what part of the ACL it can see.
```

Comment:
```suggestion
* **Henry**: We discussed this in issue 102 a bit, I'm all for changing WAC where we can to fix problems, rather than re-inventing the whole thing.
  * I definitely see that having a default rule is very useful, as it avoids copying rules all over the place.
  * I see a problem with requiring the user to be logged in before the server can determine if you or the app can see something an ACL or what part of the ACL it can see.
```